### ISSUE TITLE
Clarify autoupdate language

### ISSUE THREAD

**Issue by kingalg** [TryQuiet/quiet#1540]
Version - it's a long known issue, no specific version, but when I'm creating this task we are testing v. quiet@1.2.1-alpha.7

What happens: if the app restarted when the new version exist we update the app. Use can't do anything to prevent this update. 

What should happen: update should take place only when the user choose to update the app through our updated pop-up.

**Comment by vinkabuki** [comment]
On linux it's expected to not work according to the electron docs, they say they do not support linux at all, hence strange behavior, could you confirm that it also breaks on mac and windows?

**Comment by holmesworcester** [comment]
One solution is changing the language to clarify what has happened. I'll propose something.

**Comment by kingalg** [comment]
I'm not against forcing users to update as long as they know that it will happen. This autoupdate is mostly a problem for us because we can't get logs from terminal when it's updating. But they are some workarounds and it's not that crucial.

**Comment by kingalg** [comment]
@holmesworcester currently it looks like this:
![Screenshot 2023-06-01 at 10 38 30](https://github.com/TryQuiet/quiet/assets/68909168/07a76f37-cd8e-47b1-93ec-211bf3966dda)

**Comment by holmesworcester** [comment]
It should say:

# Update available

Quiet will upgrade to the latest version.

[Restart now]

**Comment by kingalg** [comment]
@leblowl could you provide some screenshot or other details in which version was this implemented? I don't see any changes but it may be just a matter of version. 

(btw, please don't close issues when moving them to ready for QA column because I always leave a comment in them and I need to reopen them for it anyway)

**Comment by leblowl** [comment]
@kingalg The PR in which the change was implemented (#1774) is linked to the issue here. Also in the PR is a link to the Chromatic screenshots: https://www.chromatic.com/component?appId=62021a6b9c4ef3003ab65846&csfId=components-updatemodal&buildNumber=1954&k=65034bafeb407d7a9ba69765-1200px-interactive-true&h=13&b=-1

Github automatically closes issues when tagging a PR with "fixes: #{issue-number}" (https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests#linking-a-pull-request-to-an-issue). I can perhaps just add "solution for: #{issue-number}" or something in the next PR so it stays open.

Is Chromatic sufficient or would you like an additional screenshot?

**Comment by kingalg** [comment]
@leblowl thanks for the link. I was in that PR and I saw Wiktor asking for screenshot but I didn't see any screenshot or link that's why I asked. We have old and new designes in a few places and they are changing quite often so if you go back to the issue after few weeks and it's only mentioned it can be quite challenging to find something. 

Version: 2.0.0-alpha.11 - Version: 2.0.0-alpha.12 
Implemented correctly, I saw new updater view while updating from alpha.11 to alpha.12 on Windows, MacOS and Linux.

<img width="361" alt="Screenshot 2023-09-21 at 11 11 08" src="https://github.com/TryQuiet/quiet/assets/68909168/91f41c43-9785-46f7-91f4-b4507aa5a90a">
